src/scripts/popup.js

```diff
@@ -98,7 +98,8 @@ window.PopUp = {
   renderTimer: function () {
     const rootElement = document.getElementById('root-timer');
     const entry = TogglButton.$curEntry;
-    ReactDOM.render(<Timer entry={entry} />, rootElement);
+    const project = entry ? TogglButton.findProjectByPid(entry.pid) || null : null;
```

Comment:
```suggestion
    const project = entry && TogglButton.findProjectByPid(entry.pid) || null;
```

Maybe you want to simplify it a bit